tests/engine/test_check.py

```diff
@@ -17,21 +17,25 @@ def setUp(self):
             [range(4), ["max", "min", "mean"]], names=["time", "statistic"]
         )
 
-        self.df1 = pd.DataFrame(np.random.rand(4, 12) + 1, index=index, columns=columns)
-        self.df2 = pd.DataFrame(np.random.rand(4, 12) + 2, index=index, columns=columns)
+        self.df1 = pd.DataFrame(
+            np.linspace(0.9, 1.9, 4 * 12).reshape(4, 12), index=index, columns=columns
```

Comment:
I agree, that is a nice improvement to not use random here. Maybe it would be good to have also some negative values in there for better coverage.
I would probably would have added some simplar tests here, but I guess that is out of the scope of this PR.